src/app/components/test/test.component.ts

```diff
@@ -12,10 +12,15 @@ export class TestComponent implements OnInit {
   constructor(private api: GetApiService){
   }
   ngOnInit(): void {
-    this.api.apiCall().subscribe((data:any)=>{
+    this.api.createComic("Homero", "Testing Post").subscribe((data:any)=>{
       this.activity = data.activity;
```

Comment:
```
(data:any)=>{
      console.log(data);
      this.activity = data.name;
}
```


try with this, and, when you open your browser, open dev tools (press F12 or right click anywhere in the page):
![image](https://user-images.githubusercontent.com/48605375/141656069-fdc1db31-df18-4dca-8dc8-bd1764e8a164.png)


![image](https://user-images.githubusercontent.com/48605375/141656112-e9cdb033-bd98-47be-8453-4dadcb8b8976.png)


Here you can see all `console.log();` you do in the app

Reply:
ok, done 👍